src/element/resizeElements.ts

```diff
@@ -182,14 +185,21 @@ const measureFontSizeFromWH = (
   nextHeight: number,
 ): { size: number; baseline: number } | null => {
   // We only use width to scale font on resize
-  const nextFontSize = element.fontSize * (nextWidth / element.width);
+  let width = element.width;
+
+  const hasContainer = isBoundToContainer(element);
```

Comment:
tangentially related. Can we rewrite the `isBoundToContainer` to:

![image](https://user-images.githubusercontent.com/5153846/204762841-b10c0b84-8b57-41e5-a58a-b8bc301d7653.png)

For better perf.

Also that function should be named better if it only tests for text (later we will likely have non-text container-bound elements). But that's for later.